docs/user_guide/backend_configuration.rst

```diff
@@ -40,22 +40,22 @@ To retrieve a default configuration for an in-memory ``pynwb.NWBFile`` object, u
     )
     nwbfile.add_acquisition(time_series)
 
-    default_backend_configuration = get_default_backend_configuration(
+    backend_configuration = get_default_backend_configuration(
         nwbfile=nwbfile, backend="hdf5"
     )
 
 From which a printout of the contents:
 
 .. code-block:: python
 
-    print(default_backend_configuration)
+    print(backend_configuration)
 
 returns:
 
 .. code-block:: bash
 
-    HDF5 dataset configurations
-    ---------------------------
+    HDF5 backend configuration
+    --------------------------
```

Comment:
Please move this over to https://github.com/catalystneuro/neuroconv/pull/842 so we can keep them in sync